src/pseudopeople/utilities.py

```diff
@@ -19,40 +18,37 @@
     from pseudopeople.schema_entities import DatasetSchema
 
 
-def get_randomness_stream(dataset_name: str, seed: Any, index: pd.Index) -> RandomnessStream:
-    # Avoid error from empty index
-    idx_max = max(index) if not index.empty else 1
-    map_size = max(1_000_000, idx_max * 2)
-    return RandomnessStream(
-        key=dataset_name,
-        clock=lambda: pd.Timestamp("2020-04-01"),
-        seed=seed,
-        index_map=IndexMap(size=map_size),
-    )
+def get_hash(key: str) -> int:
+    max_allowable_numpy_seed = 4294967295  # 2**32 - 1
+    return int(hashlib.sha1(key.encode("utf8")).hexdigest(), 16) % max_allowable_numpy_seed
+
+
+def get_random_generator(dataset_name: str, seed: Any) -> np.random.default_rng:
```

Comment:
We never have a use case in psp for an "additional_seed"?

Reply:
Based on how the numpy random generators work, it looks like the additional seed is not necessary since each time you use the random generator it puts you in its new state. I'm willing to change this back but it seemed nice to be able to remove some extra args.